### ISSUE TITLE
Warning: The install command is a part of the legacy v1 style of cabal usage.

### ISSUE THREAD

**Issue by dflock** [koalaman/shellcheck#2381]
When following the instructions in the README to install, I see this warning:

```console
$ cabal install ShellCheck

Warning: The install command is a part of the legacy v1 style of cabal usage.

Please switch to using either the new project style and the new-install
command or the legacy v1-install alias as new-style projects will become the
default in the next version of cabal-install. Please file a bug if you cannot
replicate a working v1- use case with the new-style commands.

For more information, see: https://wiki.haskell.org/Cabal/NewBuild
```

This is my system info:

```console
$ cabal --version

cabal-install version 2.4.0.0
compiled using version 2.4.0.1 of the Cabal library 

$ cat /etc/os-release 

NAME="Ubuntu"
VERSION="20.04.3 LTS (Focal Fossa)"
...
```

The install worked fine, it's just the warning is a little off-putting. Does the README need to be updated? I'd do it myself, but I don't know anything about cabal/haskel.
